### ISSUE TITLE
Add Reddit

### ISSUE THREAD

**Issue by ulyngs** [ulyngs/mindshield-open-source#6]
as per this review

```
My only wishes would be in depth configurations for Reddit and the ability to block Twitter replies.
```

**Comment by ulyngs** [comment]
Thanks for bringing this to my attention -- I don't actually use Redditt myself, so don't have my own intuition as to what the basic needs would be. Can you give tell me what the essential things would be that you should be able to control on Reddit, ideally with some screenshots?

**Comment by ulyngs** [comment]
@derreck503

**Comment by derreck503** [comment]
> Thanks for bringing this to my attention -- I don't actually use Reddit myself, so don't have an intuition about the basic needs there.
> 
> Can you give tell me the essential things you would like to be able to control on Reddit, ideally with some screenshots?


So my top requests 🙏 would be:
1. Hide Trending
2. Hide Feed
3. Hide Popular Communities
<img width="1436" alt="Screenshot 2023-06-04 at 2 03 29 PM" src="https://github.com/ulyngs/mindshield-open-source/assets/18196165/aaf25cc3-fcf3-4f60-beef-a6e59876c49e">

4. Hide Promoted Posts
4. Hide Related Posts sidebar
<img width="1432" alt="Screenshot 2023-06-04 at 2 04 36 PM" src="https://github.com/ulyngs/mindshield-open-source/assets/18196165/c65e1889-7eb1-47ac-b7ea-d581000f80d4">

**Comment by ulyngs** [comment]
Apologies for the delay -- I quickly tested Sune's updates on Mac and it seemed to work. I'll try to find time this week to include it and submit and update to the App Store, thanks for your patience!

**Comment by ulyngs** [comment]
Included in [c9833da](https://github.com/ulyngs/mindshield-open-source/commit/c9833da54d6d0d2da4151272f679a94e2eefc797), submitting update to the App Store now, should be out in a few days.

I don't use Reddit myself, so @adisidev and others please give feedback on whether it does what you need, and whether it should be a priority to also bring this to iOS (the update I'm submitting based on Sune's pull request is just for MacOS) :)
